var/spack/repos/builtin/packages/arm/package.py

```diff
@@ -0,0 +1,86 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+import os.path
+import re
+
+import llnl.util.tty as tty
+import spack.compiler
+import spack.util.executable
+
+
+class Arm(Package):
+    """Arm Compiler combines the optimized tools and libraries from Arm
+    with a modern LLVM-based compiler framework.
+    """
+
+    homepage = "https://developer.arm.com/tools-and-software/server-and-hpc/arm-allinea-studio"
+    url = "https://developer.arm.com/-/media/Files/downloads/hpc/arm-allinea-studio/20-2-1/Ubuntu16.04/arm-compiler-for-linux_20.2.1_Ubuntu-16.04_aarch64.tar"
+
+    # FIXME: The version is checksummed for Ubuntu 16.04, but this is not
+    # FIXME: important at the moment since the package is only meant to
+    # FIXME: provide detection
+    version('20.2.1', sha256='dc3f945b05b867809d9b507cb8ebba9cf72a6818d349207dbe1392c13dc0ad79')
```

Comment:
If the package is only meant for system detection, then we don't need a URL or version, correct?